vocabularies/survey-type.ttl

```diff
@@ -202,17 +215,20 @@ svytp:gravity-gradiometry a skos:Concept ;
     skos:broader svytp:gravity ;
     skos:definition "A variation of gravity survey where the gradient of the gravity field is measured instead of the field directly."@en ;
     skos:inScheme <http://linked.data.gov.au/def/survey-type> ;
+    skos:notation "at" ;gy

```

Comment:
remove _gy_ at end. Otherwise looks good.

Reply:
fixed to
skos:notation "gy" ;